### PR TITLE
Do not default value to placeholder

### LINKED ISSUE #29: [Bug] validate method of text take the placeholder
I tested the example in your README:

```ts
const meaning = await text({
  message: "What is the meaning of life?",
  placeholder: "Not sure",
  initialValue: "42",
  validate(value) {
    console.log('value', value)
    if (value.length === 0) return `Value is required!`;
  },
});
```

The value of the argument of validate method is the placeholder set in `text` method options (printed by the console.log)
So, you can't validate an empty response from the user

And by the way, the option `initialValue` is not available for this method and not working

### PR DESCRIPTION
**Pull request by natemoo-re**
Closes #29

### CODE REVIEW

File: examples/basic/index.ts [hunk 1]
```diff
@@ -20,15 +20,19 @@ async function main() {
...
     placeholder: "./sparkling-solid",
   });
-
+  
   if (isCancel(dir)) {
     cancel("Operation cancelled.");
     process.exit(0);
   }
+  
+  if (!dir) {
```
[the remark this answers is not in the document]
Comment (natemoo-re) [reply examples/basic/index.ts:33]: Yeah this is a good idea, but I'm not sure what the API should be. Manual fallback seems like the most explicit and flexible option for now. Can always extract some utility later.

[the remark this answers is not in the document]
Comment (ulken) [reply examples/basic/index.ts:33]: I think `default` is the convention here: [Ansible](https://www.shellhacks.com/ansible-default-variables-values/), [Inquirer](https://www.npmjs.com/package/inquirer#objects), [C#](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/default).

For consistency with other API options, `defaultValue` might be a better name.

You can still have your flexibility by not using then option and fallback to the way it works now.

I suppose one could get away with:

```ts
const result = (await text(...)) ?? "Default";
```

But might not be obvious to everyone.

[the remark this answers is not in the document]
Comment (ulken) [reply examples/basic/index.ts:33]: You even use it yourself in [Ultraflag](https://github.com/natemoo-re/ultraflag) (same thing goes for lib it's inspired by, but that may be a given).

[the remark this answers is not in the document]
Comment (natemoo-re) [reply examples/basic/index.ts]: Ah I see what you're suggesting now! Added a `defaultValue` option. 👍🏻

**Pull request merged**